src/pages/organizations.js

```diff
@@ -157,19 +132,99 @@ export const query = graphql`
               }
             }
           }
-          Sector {
+        }
+      }
+    }
+    subOrganizations: allAirtable(
+      filter: {
+        table: { eq: "Organizations" }
+        data: {
+          Categories: {
+            elemMatch: {
+              data: { Parent: { elemMatch: { id: { eq: $categoryId } } } }
+            }
+          }
+        }
+      }
+    ) {
+      nodes {
+        data {
+          Name
+          Homepage
+          About
+          Tags
+          Tagline
+          HQ_Location
+          Organization_Type
+          Headcount
+          Categories {
+            id
+            data {
+              Name
+              Parent {
+                id
+                data {
+                  Name
+                }
+              }
+            }
+          }
+          Logo {
+            localFiles {
+              childImageSharp {
+                fixed(
+                  width: 64
+                  height: 64
+                  fit: CONTAIN
+                  background: "white"
+                ) {
+                  ...GatsbyImageSharpFixed
+                }
+              }
+            }
+          }
+          LinkedIn_Profiles {
             data {
-              Slug
+              Logo {
+                localFiles {
+                  childImageSharp {
+                    fixed(
+                      width: 64
+                      height: 64
+                      fit: CONTAIN
+                      background: "white"
+                    ) {
+                      ...GatsbyImageSharpFixed
+                    }
+                  }
+                }
+              }
             }
           }
         }
       }
     }
-    sectors: allAirtable(filter: { table: { eq: "Sectors" } }) {
+    categories: allAirtable(filter: { table: { eq: "Categories" } }) {
       nodes {
+        id
         data {
-          name: Name
-          slug: Slug
+          Name
+          Count
+          Parent {
+            id
+            data {
+              Name
```

Comment:
I'm so happy we can pull arbitrary data out of the Parent link... for some reason i didnt think we could